coremltools/converters/mil/frontend/tensorflow/ops.py

```diff
@@ -2572,27 +2572,34 @@ def ResizeNearestNeighbor(context, node):
     # "ResizeNearestNeighbor" op in TF is always in the channel last mode
     # instead of upsample factor, it uses output size, which is the second input
     x = context[node.inputs[0]]
+    output_shape = context[node.inputs[1]]
 
     input_shape = x.shape  # (N,Hin,Win,C)
     if len(input_shape) != 4:
         raise ValueError('"ResizeNearestNeighbor" op: input rank is not 4')
 
-    if len(context[node.inputs[1]].shape) != 1:
+    if len(output_shape.shape) != 1:
         raise ValueError('"ResizeNearestNeighbor" op: the second input, must have rank 1')
-
-    if context[node.inputs[1]].shape[0] != 2:
+    if output_shape.shape[0] != 2:
         raise ValueError(
             '"ResizeNearestNeighbor" op: the second input, which is the output size, must have 2 elements'
         )
+
     Hout, Wout = None, None
-    if context[node.inputs[1]].val is None:
-        # for the dynamic input shape case,
-        # context[node.inputs[1]] is a mul(x=input_shape, y=scaling_factor) op.
-        scaling_factor_h = context[node.inputs[1]].op.y.val[0]
-        scaling_factor_w = context[node.inputs[1]].op.y.val[1]
+    if output_shape.val is None:
+        # The only dynamic-input-shape case that can be converted to CoreML is when
```

Comment:
I think it should be "dynamic input shape" not "dynamic-input-shape". Also should be "Core ML" not "CoreML".

This applies the comment here and the actual error message.

Reply:
revised as suggested